README.md

```diff
@@ -41,6 +41,10 @@ Or install it yourself as:
   that is used to store, by full schema name, the generated models that are
   embedded within top-level models. By default a new `Avromatic::ModelRegistry`
   is created.
+* **on_initialize**: An optional Proc that runs at the end of
+  `Avromatic.configure` and during code reloading in Rails applications. This
+  option is useful for defining models that will be extended and when the load
```

Comment:
"will be extended and when the load order is important" => "will be extended when the load is important"?